Add Sabrina sports bra (#7291)
freesewing/freesewing: sites/orgdocs/docs/designs/sabrina/readme.mdx

---
title: "Sabrina: FreeSewing's sports bra"
sidebar_label: Sabrina sports bra
---

<DesignInfo design="sabrina" />

## Designer Notes {#notes}

Sabrina is a simple pull-on sports bra, that gives light to medium support.

I have made a few as sports underwear for running to support my chest and prevent my nipples rubbing against my shirt.

Sabrina was developed from prototypes based on Breanna, but it uses a completely different construction and doesn't share any code.

If you've made Shin or Bruce or other patterns with curved seams and stretchy fabric,
Sabrina should be straightforward to make.

## What You Need {#needs}

To make Sabrina, you will need the following:

- Basic sewing supplies
- A serger (overlock machine) is highly recommended

> - Around 80 cm of suitable fabric

Reviewer: ```suggestion
- Around 0.8 metres (0.9 yards) of suitable fabric
```

- An elastic band, 2–3 cm or about an inch wide, to go around your waist

## Fabric Options {#fabric}

This pattern is designed to work with two-way-stretch fabric.

Look for a fine, soft,
comfortable synthetic sports jersey with about 20–25% spandex that stretches well in both directions.

If you don't want to use it for sports, some cotton content is fine, too.

I like to use this [Hemmers Sports jersey](https://www.fabrics-hemmers.com/sports-jersey-taupe) (available in Europe).

For prototypes, you can use some old t-shirt fabric or other inexpensive cotton jersey,
as long as it is stretchy enough.

## Cutting Instructions {#cutting}

Sabrina's body is designed to be sewn twice and then used double layered with the seams hidden between both layers.
Therefore, you need twice the fabric of what you would normally expect.

> I recommend using the same fabric for both layers,
> but if you want to use some expensive printed fabric on the outside, you can of course use different
> less expensive fabric for the lining.
> Or you can use softer, more skin-friendly fabric for the lining.
> Your choice.

Reviewer: ```suggestion
I recommend using the same fabric for both layers but you can use some expensive printed fabric on the outside whilst using less expensive fabric for the lining.
You can also use softer, more skin-friendly fabric for the lining.
Your choice.
```
Also suggest moving this too fabric options

If you prefer a thinner top, don't mind open seams or just want to create a prototype,

> you can use a single layer instead, of course.

Reviewer: ```suggestion
you can just use a single layer instead.
```
